ooniprobe/PrefixHeader.pch

```diff
@@ -67,17 +67,17 @@
 #define color_violet7 @"#4b05d4"
 #define color_violet8 @"#4d05d9"
 #define color_violet9 @"#4e05dd"
-#define color_fuschia @"#ab05cb"
-#define color_fuschia0 @"#fdeeff"
-#define color_fuschia1 @"#f8d4fe"
-#define color_fuschia2 @"#f2b5fd"
-#define color_fuschia3 @"#eb8dfc"
-#define color_fuschia4 @"#df50fa"
-#define color_fuschia5 @"#ab05cb"
-#define color_fuschia6 @"#a105bf"
-#define color_fuschia7 @"#9705b4"
-#define color_fuschia8 @"#8c04a6"
-#define color_fuschia9 @"#800498"
+#define color_fucsia @"#ab05cb"
```

Comment:
This is actually not how fuchsia is spelled either in english. See: https://en.wikipedia.org/wiki/Fuchsia_(color)